### DISCUSSION TITLE
The software keeps getting stuck on the loading screen, I am unable to use it.

### DISCUSSION THREAD

**Discussion by NomalUserC** [Griefed/ServerPackCreator#504] [Q&A]
I tried to open and use the software, but it keeps getting stuck on the loading screen, with the progress always at 80%, so I am unable to use it. I am a Minecraft player from China, and my English is not very good. This text was translated using GPT, I apologize for any inconvenience caused. I will put the screenshot below.
![Snipaste_2024-01-12_21-16-41](https://github.com/Griefed/ServerPackCreator/assets/156435805/9641273e-dbf0-49d8-899a-a4cc66e641c0)

**Comment by Griefed** [comment]
Hi NomalUserC,
could you upload the `serverpackcreator.log` from your users home-directory/ServerPackCreator/logs, please?

Cheers,
Griefed

**Reply by NomalUserC** [reply]
Thank you for your reply. I have been busy lately and I apologize for not being able to respond to you in a timely manner. The software issue seems to have been resolved, thank you very much.

**Reply by NomalUserC** [reply]
Well, I'm sorry, I just tried again to open the software and found that there is still a problem. When I tried earlier, the software was able to open normally, but now it's not working. Previously, I was able to open the software and use it normally using VPN. I think I should still attach the documents.
[logs.zip](https://github.com/Griefed/ServerPackCreator/files/13964649/logs.zip)

**Reply by NomalUserC** [reply]
[serverpackcreator.log](https://github.com/Griefed/ServerPackCreator/files/14010291/serverpackcreator.log)

**Comment by mike-brown8** [comment]
I got the same problem, and this is the log file.
[serverpackcreator.log](https://github.com/Griefed/ServerPackCreator/files/13938471/serverpackcreator.log)

This may be where the problem is (Line 198)
```log
2024-01-15T19:42:43,891 ERROR [main] (UpdateChecker.kt:48) - Either GitHub is currently unreachable, or the GitHub user/repository you set resulted in a malformed URL. PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
```

System details(partly):
|Name|Value|
|---|---|
|Program Version|5.1.11|
|System|Windows 10 IoTEnterprise(LTSC 2021) 64Bit|
|Language|Chinese Simplified|
|System Region|China Mainland|
|Time Zone|Beijing (+8)|

Best regards,
Mike

**Comment by mike-brown8** [comment]
I think this can be fixed in the `serverpackcreator-updater/src/main/kotlin/de/griefed/serverpackcreator/updater/UpdateChecker.kt` file, that  if user's system region is in China Mainland then skip update check and show a text box to tell user that update check is skipped.

**Comment by Griefed** [comment]
Heyo,

thanks for providing the logs and posting all the relevant information. A fix for your issue will be available after I'm done repairing the webservice which is currently broken, as reported in https://github.com/Griefed/ServerPackCreator/issues/485

Progress is rather slow, because work and life are rather time and energy demanding at the moment. Please be patient with me. :)

Cheers,
Griefed

**Comment by Griefed** [comment]
@mike-brown8 & @NomalUserC 
Please try [5.1.12-alpha.5](https://github.com/Griefed/ServerPackCreator/releases/tag/5.1.12-alpha.5) and see whether that works for you.

Cheers,
Griefed

**Reply by NomalUserC** [reply]
Thank you, the problem has been resolved. Version 5.1.12 can be used normally on my computer.

**Comment by Griefed** [comment]
Forgot to close the discussion. Lol.
Thanks for getting back to me. Until next time! 

Cheers,
Griefed
